tests/org.eclipse.ui.tests.navigator/src/org/eclipse/ui/tests/navigator/jst/JstPipelineTest.java

```diff
@@ -43,50 +47,43 @@ public JstPipelineTest() {
 	public void setUp() {
 		super.setUp();
 
-		WebJavaContentProvider.staticInit(_contentService
-				.getContentExtensionById(COMMON_NAVIGATOR_JAVA_EXT)
+		WebJavaContentProvider.staticInit(_contentService.getContentExtensionById(COMMON_NAVIGATOR_JAVA_EXT)
 				.getContentProvider().getClass().getClassLoader());
 	}
 
 	/*
-	 * This sort of approximates the JST/JDT pipeline relationship. The thing
-	 * this is mainly testing for is the case where the JST NCE provides JDT
-	 * objects as the content, and the JDT label provider does not get invoked
-	 * for those objects.
+	 * This sort of approximates the JST/JDT pipeline relationship. The thing this
+	 * is mainly testing for is the case where the JST NCE provides JDT objects as
+	 * the content, and the JDT label provider does not get invoked for those
+	 * objects.
 	 */
 	@Test
 	public void testJstPipeline() throws Exception {
 
-		_contentService.bindExtensions(new String[] {
-				COMMON_NAVIGATOR_RESOURCE_EXT, COMMON_NAVIGATOR_JAVA_EXT,
-				TEST_CONTENT_JST }, false);
+		_contentService.bindExtensions(
+				new String[] { COMMON_NAVIGATOR_RESOURCE_EXT, COMMON_NAVIGATOR_JAVA_EXT, TEST_CONTENT_JST }, false);
 
 		// Note this test will fail showing only one if the JDT stuff
 		// is not included in the executing bundles (which it normally is)
-		assertEquals(
-				"There should be two visible extensions for the pipeline viewer.",
-				3, _contentService.getVisibleExtensionIds().length);
+		assertEquals("There should be two visible extensions for the pipeline viewer.", 3,
```

Comment:
I know this is old, but "two visible extension" vs using 3 sounds something we should adjust.